### DISCUSSION TITLE
Constant defined by the optimisation process

### DISCUSSION THREAD

**Discussion by lcgfccr** [cvxpy/cvxpy#1473] [Q&A]
Hello,

I was wondering if there is a way with CVXPY to solve a convex minimisation problem that implies a constraint with a unique constant `k` whose value should be determined during the optimisation process as the one that grants the minimisation of the objective function. 

I cannot declare it as `Variable` because it is not a variable of the problem like `y` nor as `Constant` because in such case its value has to be given in input.

Please consider the following picture. To be thorough, `x` and `b` are `(1,n)` vectors, `C` is `(n,n)` positive definite matrix, and `i` is a `(1,n)` vector of ones.
<img width="578" alt="2" src="https://user-images.githubusercontent.com/15036808/132731063-2804d0f7-c4a7-4a57-a194-9333145406b6.png">

**Comment by SteveDiamond** [comment]
Hi @poshlf, I don't quite understand your question. You can of course solve the problem for any fixed k. What do you mean by it's a constant that you optimize over but not a variable? Do you want to maximize over k?

**Reply by rileyjmurray** [reply]
Checking in, @poshlf. Could you provide more information?
